pkg/clusterresource/controller.go

```diff
@@ -239,14 +239,14 @@ func (c *controller) syncNamespace(ctx context.Context, namespace NamespaceName,
 		templateFileName := templateFile.Name()
 		if filepath.Ext(templateFileName) != ".yaml" {
 			// nothing to do.
-			logger.Infof(ctx, "syncing namespace [%s]: ignoring unrecognized filetype [%s]",
+			logger.Debugf(ctx, "syncing namespace [%s]: ignoring unrecognized filetype [%s]",
```

Comment:
is this not a warning? We are ignoring something?

Reply:
not necessarily, we only look at yaml files (because of how kubernetes mounts configmaps with symlinks this was causing problems in the sandbox mode)